lib/recaptcha/token.rb

```diff
@@ -0,0 +1,19 @@
+module Recaptcha
+  module Token
+    require 'json'
+
+    def self.secure_token
+      private_key  =  Recaptcha.configuration.private_key
+      raise RecaptchaError, "No private key specified." unless private_key
+
+      stoken_json = {'session_id' => SecureRandom.uuid, 'ts_ms' => (Time.now.to_f * 1000).to_i}.to_json
```

Comment:
We're using it in tandem, which in hindsight explains why I blatantly overlooked this. We have a SSO domain which uses `<%= recaptcha_tags %>` (hence the gem), and then a ton of other domains which use just the token.